lnd.go

```diff
@@ -667,25 +667,54 @@ func Main(cfg *Config, lisCfg ListenerCfg, implCfg *ImplementationCfg,
 	ltndLog.Infof("Waiting for chain backend to finish sync, "+
 		"start_height=%v", bestHeight)
 
+	type syncResult struct {
+		IsSynced bool
+		TS       int64
+		Err      error
+	}
+
+syncCheckLoop:
```

Comment:
nit: would personally prefer not using `syncCheckLoop` and just do `break`

Reply:
Thanks! The reason for the labelled loop is that just using a `break` would've just cancelled the `select` statement and not the outer loop.

I refactored this a bit though so that just canceling the `select` statement works fine :)! I also prefer the new version without the `syncCheckLoop ` label, as it makes it a bit clearer what the actual loop is for.